tests/drivers/build_all/interrupt_controller/intc_plic/app.edge_interrupt.overlay

```diff
@@ -7,7 +7,7 @@
  /{
 	soc {
 		plic: interrupt-controller@c000000 {
-			riscv,trigger-reg-offset = < 0x00001080 >;
+			support-edge-interrupt;
```

Comment:
possibly remove, depending on what the consensus is.

Reply:
The parameter is dropped in #63273